go.mod

```diff
@@ -78,18 +79,18 @@ require (
 	go.uber.org/zap v1.19.0
 	golang.org/x/crypto v0.1.0
 	golang.org/x/net v0.4.0
-	golang.org/x/oauth2 v0.0.0-20220223155221-ee480838109b
+	golang.org/x/oauth2 v0.0.0-20220822191816-0ebed06d0094
```

Comment:
I expected to bump to a version of `golang.org/x/oauth2` (v0.2.0+) that used the new isolated `cloud.google.com/go/compute/metadata` (https://github.com/golang/oauth2/blob/v0.2.0/go.mod#L6) to avoid needing to pull in more of `cloud.google.com/go` and `cloud.google.com/go/compute`

Reply:
Those were the bumps tied to the go-oidc bump; I can address oauth2 separately. I’ve opened https://github.com/coreos/go-oidc/pull/361 for go-oidc itself.